coreneuron/nrnoc/multicore.cpp

```diff
@@ -67,7 +68,7 @@ the handling of v_structure_change as long as possible.
 
 namespace coreneuron {
 #define CACHELINE_ALLOC(name, type, size) \
-    name = (type*)nrn_cacheline_alloc((void**)&name, size * sizeof(type))
+    name = (type*)emalloc_align(size * sizeof(type), NRN_SOA_BYTE_ALIGN)
```

Comment:
this `CACHELINE_ALLOC` is used at only one place i.e. multicore.cpp. Could you replace this macro by actual definition?